packages/http-client-csharp/eng/scripts/Add-Debug-Profile.ps1

```diff
@@ -14,22 +14,22 @@
 .PARAMETER SdkDirectory
     Path to the target SDK service directory
 
-.PARAMETER Generator
-    Generator name (default: ScmCodeModelGenerator)
+.PARAMETER UseMgmt
```

Comment:
We can get rid of this parameter. We should just infer mgmt or not based on the sdkPath. Specifically, if the leaf directory has "ResourceManager" in the name then UseMgmt is true.